app/user/rest.py

```diff
@@ -667,6 +668,12 @@ def update_password(user_id):
     return jsonify(data=user.serialize()), 200
 
 
+@user_blueprint.route("/report-all-users", methods=["GET"])
+def report_all_users():
+    users = dao_report_users()
+    return jsonify(data=users.serialize()), 200
```

Comment:
I see this is following the same pattern as a bunch of the other endpoints, so I'm assuming the frontend would have to take the JSON and convert it into a CSV or whatever else we'd like to do with it?

Reply:
We caught up and yes, the frontend will have to account for this endpoint.